### PR TITLE
default values

### LINKED ISSUE #4: Default values
The instatiation of a `KeycloakAdmin` or `KeycloakOpenID` require the definition (via environment variable or argument) of at least:
- `server_url`
- `realm_name`
- `client_id`
- `client_secret_key`

In standard deployments, some defaults can be assumed for certain values:
- `server_url`: _http://keycloak:8080/auth/_
- `client_id`: _shieldapi_

### LINKED ISSUE #5: client_secret_key consistency
For consistency, client_secret_key should be renamed to client_secret

### PR DESCRIPTION
**Pull request by pablo-de-andres**
Also closes #5

### CODE REVIEW

**Comment by pablo-de-andres** [comment]
Also closes #5

**Review by yoavnash — APPROVED** [review]

File: shieldapi/keycloak_utils.py [hunk 1]
```diff
@@ -85,58 +79,54 @@ def get_keycloak_admin(
...
+       UserWarning: If both the parameter and env-variable are set.
 
-       ValueError: If both variables are `None` or if the Python-object is `False`
-       and env-variable not `None`.
+       ValueError: If both the parameter and env-variable are None.
     """
     env = os.environ.get(env_var)
-    if var and env:
-        message = f"""Both the kwarg and the env-variable for `{env_var}` are assigned.
+    if (param or isinstance(param, bool)) and env:
```
Comment (MBueschelberger) [inline shieldapi/keycloak_utils.py:125]: Okay, this code makes the retrieval of the value more effective. Nice :)

File: shieldapi/keycloak_utils.py [hunk 2]
```diff
@@ -40,14 +36,16 @@ def get_keycloak_openid(
...
     return KeycloakOpenID(
-        server_url=_get_value(server_url, "KEYCLOAK_HOST"),
+        server_url=_get_value(
+            server_url, "KEYCLOAK_HOST", "http://keycloak:8080/auth/"
+        ),
         realm_name=_get_value(realm_name, "KEYCLOAK_REALM_NAME"),
-        client_id=_get_value(client_id, "KEYCLOAK_CLIENT_ID"),
-        client_secret_key=_get_value(client_secret_key, "KEYCLOAK_CLIENT_SECRET_KEY"),
+        client_id=_get_value(client_id, "KEYCLOAK_CLIENT_ID", "shieldapi"),
+        client_secret_key=_get_value(client_secret, "KEYCLOAK_CLIENT_SECRET"),
```
Comment (MBueschelberger) [inline shieldapi/keycloak_utils.py:48]: Why do you go back to `client_secret` when the original kwarg from Python is `client_secret_key` ? I think it makes sense for a user to keep the name of the kwarg as close as it is from the original function.

Comment (pablo-de-andres) [reply shieldapi/keycloak_utils.py:48]: The `key` is added by the python keycloak library, Keycloak itself also uses [`client_secret`](https://www.keycloak.org/docs-api/18.0/rest-api/index.html#_getclientsecret)

Comment (MBueschelberger) [reply shieldapi/keycloak_utils.py:48]: Okay, that's odd. Nervermind

**Review by MBueschelberger — APPROVED** [review]

**Pull request merged**